js/index.js

```diff
@@ -70,7 +70,7 @@ function getFrameText(frames, currentTime) {
 
 
 function initializeVideoMap(data) {
-    const baseUrl = data.base_url || '';
+    const baseUrl = DATA_URL.split("videos.geojson")[0];
```

Comment:
hmm I don't love the fact that this enforces that the file be called `videos.geojson` :(

Seems okay for now, but a bit brittle. I think this is fine for now, but if we're doing anything serious with this, let's think about how to improve this a bit.